bio/seqtk/subsample/pe/environment.yaml

```
@@ -3,3 +3,4 @@ channels:
   - conda-forge
 dependencies:
   - seqtk ==1.3
+  - pigz ==2.3.4
```

Maybe unpin the patch version, to allow using newer patch versions that might have bugs removed or even security problems patched. Or is there a particular need for this exact version?
```suggestion
  - pigz =2.3
```
Accordingly, for the `se` wrapper.

---

Nope, minor version should suffice.

---

Done in c62ca19.